src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService service = new UserService();
+        return service.findByEmail(email) != null
```

Comment:
Can you avoid calling findByEmail twice? 

Let's create local variable for the result of this metho